### PR TITLE
Support background color of WebView. Close #1811

### LINKED ISSUE #1811: WebView doesn't support backgroundColor in the style property
### Environment

1. `react-native -v`:
react-native-cli: 2.0.1
react-native: 0.55.3

2. `npm ls rnpm-plugin-windows`:
`-- rnpm-plugin-windows@0.2.8

3. `npm ls react-native-windows`:
`-- react-native-windows@0.55.0-rc.0

4. `node -v`:
v9.11.1

5. `npm -v`:
5.6.0

6. `yarn --version`:
1.6.0

Then, specify:

- Target Platform: UWP

- Target Platform Version(s): UWP (10.0.14393)

- Target Device(s): Desktop PC (Windows 10)

- Development Operating System: Windows 10

- Visual Studio Version: Visual Studio 2017

### Steps to Reproduce

1. Use `<WebView style={{ backgroundColor: 'transparent' }} ... >` in code

### Expected Behavior

The background color of the WebView is transparent.

### Actual Behavior

The background color of the WebView is white.

### Reproducible Demo

I'll prepare the demo if needed.

### PR DESCRIPTION
**Pull request by exKAZUu**
Close #1811

### CODE REVIEW

File: ReactWindows/ReactNative/Views/Web/ReactWebViewManager.cs [hunk 1]
```diff
@@ -76,6 +77,24 @@ public override JObject ViewCommandsMap
...
+        /// <param name="view">The view instance.</param>
+        /// <param name="color">The masked color value.</param>
+        [ReactProp(ViewProps.BackgroundColor, CustomType = "Color")]
+        public void SetBackgroundColor(WebView view, uint? color)
+        {
+            if (color.HasValue) {
+                var parsedColor = ColorHelpers.Parse(color.Value);
+                // "style={{ backgroundColor: 'transparent' }}" gives #00000000,
+                // but we need #00FFFFFF to make the background color transparent
+                view.DefaultBackgroundColor = parsedColor.A == 0 ? Colors.Transparent : parsedColor;
```
Comment (rozele) [inline ReactWindows/ReactNative/Views/Web/ReactWebViewManager.cs]: I'm curious why this is inconsistent with the background color in ReactViewManager. Why do we only need to apply this change here?

**Comment by exKAZUu** [comment]
@matthargett (c.c. @rozele) There seems no way to change the background color in WPF (c.f. https://stackoverflow.com/questions/20853441/wp-webbrowser-black-background).
We can still modify a loaded html like injecting `"background-color:black"` into the `sytle` attribute of `html` element, but obviously this functionality is different from `DefaultBackgroundColor` in UWP.

File: ReactWindows/ReactNative/Views/Web/ReactWebViewManager.cs (hunk 1 above)
Comment (exKAZUu) [reply ReactWindows/ReactNative/Views/Web/ReactWebViewManager.cs]: Hmm, right. Though I think this modification should be applied all the places where `react-native-windows` uses colors, this PR should be simple. So, I will remove this modification.

**Comment by exKAZUu** [comment]
@rozele I modified the code to deal with your comments. Thanks!

**Pull request merged**